lib/components/cards/VeggieCardSeasons.dart

```diff
@@ -15,9 +15,9 @@ class VeggieCardSeasons extends StatelessWidget {
       children: [
         for (Season season in veggie.seasons)
           Padding(
-            padding: size == Size.large
-                ? EdgeInsets.only(left: $styles.padding.s)
-                : EdgeInsets.only(left: $styles.padding.xs),
+            padding: EdgeInsets.only(
```

Comment:
Tiny simiplification.

Reply:
I can't believe I hadn't even considered this as possible!